Update CFP and PC for TCRS '24 (#10)
tcrs-ws/tcrs-ws.github.io: cfp.md

- Time-predictable hardware

More background about this workshop and its motivation can be found [here]({% link about.md %}).

## Important Dates
- July 8, 2024: Paper submission deadline
- Aug 5, 2024: Author notification for the first round of reviews
- Aug 12, 2024: Camera-ready and video graphical abstract deadline
- Aug 19, 2024: Final acceptance notification

> - TBD (One day during ESWEEK, the week of September 29 - October 4, 2024): Paper presentations

Reviewer: wasn't this set to Thursday or Friday now?

Author: Thanks for pointing this out. I updated this with more specific tentative dates (Thu, Oct 3 or Fri, Oct 4), although it's not finalized yet.

- Time-predictable hardware

More background about this workshop and its motivation can be found [here]({% link about.md %}).

## Important Dates
- July 8, 2024: Paper submission deadline
- Aug 5, 2024: Author notification for the first round of reviews

> - Aug 12, 2024: Camera-ready and video graphical abstract deadline

Reviewer 2: What does this mean?

Reviewer 2: I read on and now know what it means, but I think it would be better to specify this earlier. Maybe before the Important Dates, we should have a section that outlines the different tracks, and what authors should expect to deliver (i.e., graphical abstract for IEEE ESL). We can then go into the details of the logistics further down the page.

Author: > we should have a section that outlines the different tracks,

We have only a single track (for submission) but just dual acceptance tiers.

> what authors should expect to deliver (i.e., graphical abstract for IEEE ESL)

This is only needed for the papers that are accepted for the journal publication. No need to submit graphical abstract for the initial submission. Maybe we should be clearer about this.

> We can then go into the details of the logistics further down the page.

Thanks. Let me work on this.

Author: I reorganized the content.

Reviewer 2: This looks good!

TCRS '24 will have two tiers of final acceptance.

1. *Workshop-presentation* acceptance.
2. *Publication-and-Workshop-presentation* acceptance.

The *Workshop-presentation* tier acceptance will be given to the submissions that meet the TCRS workshop quality requirements.
The *Publication-and-Workshop-presentation* tier acceptance will be given to the submissions that meet the quality requirements for IEEE ELS journal publication as well as the quality requirements for the TCRS workshop.

All papers with any of these two tiers of acceptance will be presented at the workshop and distributed as proceedings of the workshop.
However, only the papers with the *Publication-and-Workshop-presentation* tier acceptance will be publisehd in IEEE ESL special issue of TCRS.

### Review Process
Authors of the submissions will be notified with the first round of reviews.
Below are the subsequent processes for accepted papers.

1. Authors of the submissions with *Workshop-presentation* acceptance will be asked to submit the final PDFs that will be included in the "*proceedings to distribute*."

> The review result from IEEE ESL will be a "reject" for the submission with *Workshop-presentation* acceptance; however, the authors will receive separate notices for *Workshop-presentation* acceptance.

Reviewer 2: I'm having a hard time following this. Are there two different kinds of acceptance? Present-only and Present-and-publish? The text is a bit unclear...

Author: I reworded the tiers!

### Acceptance Tiers
At least one of the authors for each paper must fully register and present the accepted paper at the workshop.
TCRS '24 will have two tiers of final acceptance.

1. *Present-only* acceptance.
2. *Present-and-publish* acceptance.

The *Present-only* tier acceptance will be given to the submissions that meet the TCRS workshop quality requirements.
The *Present-and-publish* tier acceptance will be given to the submissions that meet the quality requirements for IEEE ELS journal publication as well as the quality requirements for the TCRS workshop.

All papers with any of these two tiers of acceptance will be presented at the workshop and distributed as proceedings of the workshop.
However, only the papers with the *Present-and-publish* tier acceptance will be published in the IEEE ESL special issue of TCRS.

### Review Process
Authors of the submissions will be notified with the first round of reviews.
Below are the subsequent processes for accepted papers.

1. Authors of the submissions with *Present-only* acceptance will be asked to submit the final PDFs that will be included in the "*proceedings to be distributed*."

> The review result from IEEE ESL will be a "reject" for the submission with *Present-only* acceptance; however, the authors will receive a separate notice for *Present-only* acceptance.

Reviewer 2: I don't understand this sentence.

Reviewer 2: ```suggestion
Submissions with a *Present-only* acceptance will not qualify for publication with IEEE ESL.
```

Author: Your suggested change actually changes the intended message... They will indeed receive a "reject" decision from the journal but will receive a separate notice of acceptance for presentation. What parts of the sentence do you not understand? I think the message is clear.

Reviewer 2: It's just a confusing sentence. Accept == reject? What I did was simplify the sentence and focus on the result: present-only means no ESL. How is that not clear?

> 2. Authors of the submissions with *Present-and-publish* acceptance will receive a "minor revision" decision from IEEE ESL and will be required to submit the final camera-ready PDF and a video graphical abstract which is a short video presenting the paper.

Reviewer 2: ```suggestion
2. Authors of the submissions with *Present-and-publish* acceptance will be expected to incorporate the review feedback into a minor revision that is to be submitted for publication to IEEE ESL. In addition to submitting the final camera-ready version of their paper, authors are asked to submit a video graphical abstract (which is a short video presenting the paper).
```

Author: I think the first part of your suggested change is not accurate. They will indeed receive a "minor revision" decision from the journal. Let me come up with a better wording...

Reviewer 2: Does it matter how the details of the interaction with the journal play out?

Author: Yes, I think it matters to avoid confusion. For example, if authors receive "reject" notice they may think they won't be able to present their paper at the workshop either.

Both the final PDF manuscript and the video graphical abstract will be published by IEEE ESL.
See [here](https://ieee-ceda.org/publication/esl/esl-paper-submission) for more information on the video graphical abstract.

> Final submissions with *Present-and-publish* acceptance, including both the final camera-ready PDF and the video graphical abstract, will be reviewed in the second round.

Reviewer 2: No I'm confused again. Camera-ready instructions are above, but now we're talking about a second round of reviews?

Author: Maybe we should not call it camera-ready, then. The second submission is indeed subject to a second round of reviews. Let me work on this a bit.

The review process is **blind** (not double-blind);
please do not submit anonymized manuscripts.

### Submission Site
Please submit your paper at the [Manuscript Central Submission Site](https://mc.manuscriptcentral.com/les-ieee).
Click on "**Author**," create a new manuscript, and select "**Type**" as "**SI:TCRS24**".

### Conflicts of Interest Requirements
In addition to the manuscript, the authors are required to submit a **Cover Letter** indicating the **Conflicts of Interest (COIs)** with the program committee members through Manuscript Central.
The authors can submit the cover letter during the Submission **Step 6: Details & Comments**, using a textbox under **Write Cover Letter**.
Even if the authors do not have any COIs, they are still required to submit a cover letter confirming that they have no COIs.

### Acceptance Tiers
At least one of the authors for each paper must fully register and present the accepted paper at the workshop.
TCRS '24 will have two tiers of final acceptance.

1. *Present-only* acceptance.
2. *Present-and-publish* acceptance.

The *Present-only* tier acceptance will be given to the submissions that meet the TCRS workshop quality requirements.

> The *Present-and-publish* tier acceptance will be given to the submissions that meet the quality requirements for IEEE ELS journal publication as well as the quality requirements for the TCRS workshop.

Reviewer 2: ```suggestion
The *Present-and-publish* tier acceptance will be given to the submissions that are accepted for presentation at the TCRS workshop _and_ considered for publication in the IEEE ELS journal.
```

(acceptance is not only based on quality but also fit/topic, etc.)

All papers with any of these two tiers of acceptance will be presented at the workshop and distributed as proceedings of the workshop.
However, only the papers with the *Present-and-publish* tier acceptance will be published in the IEEE ESL special issue of TCRS.

### Review Process
Authors of the submissions will be notified with the first round of reviews.
Below are the subsequent processes for accepted papers.

#### 1. *Present-only* acceptance
Authors of a submission with *Present-only* acceptance will be asked to submit the final version of their paper, which will be included in the "*proceedings to be distributed*."

> Although the official review result from the IEEE ESL journal will be a "reject," the authors will receive a separate notice for *Present-only* acceptance.

Reviewer 2: This seems to suggest that all submissions that are "reject" will be accepted as "present-only". I don't think this is what we mean.

I would word it differently, How about:
"Papers that not accepted for publication in the IEEE ESL journal may still be eligible to present at the workshop. In this case, they will receive a "reject" from IEEE ESL and a separate "Present-only" acceptance notification from TCRS."

```suggestion
Even if the official review result from the IEEE ESL journal is "reject," the authors may still receive a separate notice for *Present-only* acceptance.
```

Author: Thanks! I changed it as you suggested.
